app/screens/thread/thread.tsx

```diff
@@ -80,7 +81,10 @@ const Thread = ({
     }, [componentId, rootId, isCRTEnabled]);
 
     useEffect(() => {
+        storeLastViewedThreadIdAndServer(rootId);
+
         return () => {
+            removeLastViewedThreadIdAndServer();
```

Comment:
Is this not called when the app closes?

Reply:
No, it's only called when user leaves thread screen